acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java

```diff
@@ -130,6 +130,7 @@ public boolean updateAccessConfig(PlainAccessConfig plainAccessConfig) {
             log.error("Parameter value plainAccessConfig is null,Please check your parameter");
             throw new AclException("Parameter value plainAccessConfig is null, Please check your parameter");
         }
+        buildPlainAccessResource(plainAccessConfig); // check config
```

Comment:
It seems that only null checks is necessary here, extracting a check method to accomplish it may be better.

Reply:
Yes, I agree with above suggestion!

Reply:
Fixed. Thanks for your advice.